client/src/lib/restService.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
```

Comment:
Generally there is separate place for http calls with function dedicated  point to specific endpoint or some function that help with sending authentication credentials. In this situation I don't see much difference between using the functions you made over using axios.get or axios.post. Yes,  It's good practice for the future its better because we could add functionality  like adding authentication credentials for each request or something like this.